### PR TITLE
Add Bamboots

### PR DESCRIPTION
**Pull request by arasu01**
Bamboots

## Project URL
https://github.com/mmoaay/Bamboots

## Category
Networking

## Description
Bamboots is a network request framework based on Alamofire, aiming at making network request easier for business development.
 
## Checklist

- [x] Only one project/change is in this pull request
- [x] Addition in chronological order (bottom of category)
- [x] Supports iOS 9 / tvOS 10 or later
- [ ] Supports Swift 4 or later
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 2 link issues, a project collaborator will take care of these, thanks :)</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
736 | [301](https://httpstatuses.com/301) | https://github.com/knly/Evergreen redirects to<br>https://github.com/nilsleiffischer/Evergreen
2878 | [404](https://httpstatuses.com/404) | https://www.manning.com/books/ios-development-with-swift 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -1042,6 +1042,7 @@ Also see [push notifications](#push-notifications)
 . :large_orange_diamond:
 * [Ciao](https://github.com/AlTavares/Ciao) - Publish and discover services using mDNS(Bonjour, Zeroconf). :large_orange_diamond:
 * [PerfectAPIClient](https://github.com/SvenTiigi/PerfectAPIClient) - An API Client based on a network abstraction layer for the Perfect Server-Side Swift Framework 🔶
+* [Bamboots](https://github.com/mmoaay/Bamboots) - Bamboots is a network request framework based on Alamofire, aiming at making network request easier for business development.
```
Comment (lfarah) [inline README.md]: Please add the :large_orange_diamond:

**Comment by arasu01** [comment]
@lfarah: Request changes are done.

**Comment by lfarah** [comment]
Thanks!

**Pull request merged**